### PR TITLE
Updated to catch expired tokens

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

File: src/js/lib/handleLoggedInStatus.js [hunk 1]
```diff
@@ -2,15 +2,21 @@ var handleLoggedInStatus = (function () {
...
-    var template = Handlebars.compile($('#logged-in-template').html());
-    $('#navbar-tools').html(template(data));
-    data.role.forEach(function(role) {
-      if (role.id === 'admins') {
-        var adminTemplate = Handlebars.compile($('#admin-menu-template').html());
-        $('#admin-menu').html(adminTemplate());
+    return new Promise(function (resolve, reject) {
+      var template = Handlebars.compile($('#logged-in-template').html());
+      $('#navbar-tools').html(template(data));
+      if (!data.role) {
```
Comment (Lukewh) [inline src/js/lib/handleLoggedInStatus.js]: If data.role is not set, the token is invalid - so kick them.
Previously the code just stopped at this point so wouldn't show any button

**Comment by philmbst** [comment]
LGTM

File: src/js/lib/atlasUser.js [hunk 2]
```diff
@@ -9,18 +9,21 @@ var atlasUser = (function () {
...
-        iPlanetDirectoryPro: authCookie
-      },
-      success: function (data) {
-        callback(data);
-      },
-      error: function (jqXHR, textStatus, errorThrown) {
-        log.error(errorThrown);
-      }
+  var getUserData = function (url, authCookie) {
+    return new Promise(function (resolve, reject) {
```
Comment (devo132) [inline src/js/lib/atlasUser.js]: $.ajax() already returns a promise, unless there's a need for it to be an ES6 promise specifically

Comment (Lukewh) [inline src/js/lib/atlasUser.js]: For consistency with the other Promise I added so as not to mix Promise libraries (I'm not sure how the jQuery promise library behaves)

**Comment by devo132** [comment]
LGTM 🔢

**Pull request merged**